libs/platform/src/lib/components/table/table.component.spec.ts

```diff
@@ -2,19 +2,12 @@ import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
 import { Observable, of } from 'rxjs';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 
-import {
-    DialogModule,
-    ListModule,
-    PopoverModule,
-    RtlService,
-    TableModule,
-    CheckboxModule
-} from '@fundamental-ngx/core';
+import { ListModule, PopoverModule, RtlService, TableModule, CheckboxModule } from '@fundamental-ngx/core';
 
 import { TableComponent } from './table.component';
-import { CollectionStringFilterStrategy, SelectionMode, SortDirection } from './enums';
+import { FILTER_STRING_STRATEGY, SelectionMode, SortDirection } from './enums';
 import { TableDataProvider, TableDataSource } from './domain';
-import { TableState } from './interfaces';
+import { CollectionFilter, CollectionGroup, CollectionStringFilter, TableState } from './interfaces';
 import { TableService } from './table.service';
```

Comment:
It would be good to add some basic unittests also testing not only programatic side of it and its internal but also with template where you add basics bindings. and tests expected result.

Reply:
@fkolar I added unit tests where table is driven by template. Please take a look